packages/main/src/components/user-organisations-listing/UserOrganisationsListing.jsx

```diff
@@ -141,8 +145,12 @@ const Organization = styled.div`
 const Logo_Members = styled.div`
   display: flex;
   flex-direction: column;
+<<<<<<< HEAD
+  justify-content: space-between;
+=======
   justify-content: space between;
   width: 100%;
+>>>>>>> 18cdd3f0e4f4bac88c71d5e5bcbfa1d43de17fa2
```

Comment:
Fix these conflicts please. the `justify-content: space-between;` is repeated twice and remove this `<<<<<<< HEAD` and  this `>>>>>>> 18cdd3f0e4f4bac88c71d5e5bcbfa1d43de17fa2`

Reply:
Removed already @JulianaSau